docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,35 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's encouraged to wait for the transactions to be mined and to check how they appear on the chain.
+
+#### Lido CLI
+
+Make sure Python with pip is installed and then run:
+
+```sh
+pip install lido-cli
+lido-cli --rpc http://1.2.3.4:8545 validate_network_keys
+```
+
+This operation checks every Lido key for validity and thus is very CPU-intensive and takes a long time to process the keys.
```

Comment:
"Note that the check is a CPU-heavy operation and could take some time to process the keys."
If we may give the ballpark for the time of check (minutes, hours?) — would be great